README.md

```diff
@@ -137,8 +137,8 @@ anywhere.
 In other words, kube-state-metrics itself is designed to be another source for
 Heapster (although this is not currently the case).
 
-Additionally, some monitoring systems such as Prometheus do not use Heapster
-for metric collection at all and instead implement their own. Having
+Additionally, some monitoring systems such as Prometheus can use both Heapster
+and kube-state-metrics as scrape target for metric collection. Having
```

Comment:
@brancz but here the heapster does mention that it can support prometheus https://github.com/kubernetes/heapster/blob/master/docs/debugging.md#debuging , comments?

```
`/metrics` contains lots of metrics in Prometheus format that can indicate the root cause of Heapster problems. Example:
```